app/assets/javascripts/application.js

```diff
@@ -1,5 +1,2 @@
-//= require jquery
-//= require jquery_ujs
 //= require turbolinks
-//= require_directory ./app
-//= require_self
+//= require app/utils
```

Comment:
Why switch from require_directory?   If we don't want to use 'app' directory to group application fragments then why even have the 'app' sub-directory at all?